modules/component/src/componentNext.ts

```diff
@@ -69,7 +33,17 @@ type Comparator = <S, P>(s1: S, p1: P, s2?: S, p2?: P) => boolean
 const defaultComparator: Comparator = () => true
 
 const arg2 = <A, B>(a: A, b: B) => b
+
+/**
+ * Type safe, composable, and view library agnostic component API
```

Comment:
-  hyphenate `type-safe` and `view-library`
- frontload or remove the word `API`, so that it can be implied

Something like:
> Type-safe and composable Components that are agnostic of view-library